src/widget/geo/geopicker.js

```diff
@@ -40,15 +40,15 @@ let searchSource =
     'https://maps.googleapis.com/maps/api/geocode/json?address={address}&sensor=true&key={api_key}';
 const googleApiKey = config.googleApiKey || config.google_api_key;
 const iconSingle = L.divIcon({
-    iconSize: 24,
+    iconSize: [16, 24],
```

Comment:
This does two things:

1. Correct the width of map pins
2. Prevent an error when clicking on the pin, where Leaflet's internal state calculating the pin's offset has `NaN` for the left side.